SConstruct

```diff
@@ -1,4 +1,46 @@
 # -*- python -*-
+import os
 from lsst.sconsUtils import scripts
+from lsst.sconsUtils.state import env
+from lsst.sconsUtils.utils import libraryLoaderEnvironment
+from SCons.Script import Default
+
 # Python-only package
-scripts.BasicSConstruct("drp_pipe", disableCc=True, noCfgFile=True)
+# Force shebang and policy to come first so the file first appears in the bin
+# directory before it is used. This is required to run on macos.
+targetList = (
+    "version",
+    "shebang",
+    "policy",
+) + scripts.DEFAULT_TARGETS
+scripts.BasicSConstruct(
+    "drp_pipe", disableCc=True, noCfgFile=True, defaultTargets=targetList
+)
+PKG_ROOT = env.ProductDir("drp_pipe")
+
+# make deepCoadd injection pipelines for rc2_subset and RC2
+rc2_subset_injected_deepCoadd = env.Command(
+    target=os.path.join(
+        PKG_ROOT, "pipelines", "HSC", "DRP-RC2_subset+injected_deepCoadd.yaml"
+    ),
+    source=os.path.join(PKG_ROOT, "pipelines", "HSC", "DRP-RC2_subset.yaml"),
+    action=" ".join(
+        [
+            libraryLoaderEnvironment(),
+            "make_injection_pipeline -t deepCoadd -r $SOURCE -f $TARGET --overwrite",
+        ]
+    ),
+)
+RC2_injected_deepCoadd = env.Command(
```

Comment:
As a variable, this should probably have been `rc2_injected_deepCoadd`. Apologies for not catching it before the merge. It's not a serious issue, but something one of us can address in a future ticket next time we're poking around in this package.